src/agent/src/mount.rs

```diff
@@ -233,6 +233,15 @@ async fn ephemeral_storage_handler(
     Ok("".to_string())
 }
 
+#[instrument]
+async fn overlayfs_storage_handler(
+    logger: &Logger,
+    storage: &Storage,
+    sandbox: Arc<Mutex<Sandbox>>,
```

Comment:
The compiler should complain about this unused parameter.

Reply:
done